crates/air_r_formatter/src/r/auxiliary/call_arguments.rs

```diff
@@ -944,20 +945,53 @@ impl Format<RFormatContext> for FormatAllArgsBrokenOut<'_> {
             Ok(())
         });
 
+        let args = if !self.expand && is_hugging_call(self.args)? {
+            Either::Left(args)
+        } else {
+            Either::Right(soft_block_indent(&args))
+        };
+
         write!(
             f,
             [group(&format_args![
                 self.l_token,
                 format_with(|f| f.join().entries(self.leading_holes.iter()).finish()),
                 maybe_space(!self.leading_holes.is_empty() && !self.args.is_empty()),
-                soft_block_indent(&args),
+                &args,
                 self.r_token,
             ])
             .should_expand(self.expand)]
         )
     }
 }
 
+fn is_hugging_call(items: &[FormatCallArgument]) -> SyntaxResult<bool> {
+    // We only consider calls with one argument
+    let Some(item) = items.first() else {
+        return Ok(false);
+    };
+    if items.len() != 1 {
+        return Ok(false);
+    }
+
+    // Unwrap the value to get the `AnyRExpression`
+    let Some(arg) = item.element().node.clone()?.value() else {
```

Comment:
```suggestion
    let Some(arg) = item.element().node()?.value() else {
```

Reply:
The clone is necessary

Reply:
I can make it work with:

```rust
    let Some(arg) = item.element().node.as_ref().ok() else {
        return Ok(false);
    };

    let Some(arg) = arg.value() else {
        return Ok(false);
    };
```

but I'm not sure that's a win